source/ceylon/time/Date.ceylon

```diff
@@ -32,6 +32,15 @@ shared interface Date
 
     "Returns the [[DateRange]] between this and given [[Date]]."
     shared formal DateRange rangeTo( Date other );
+    
+    "Two `Date`s are considered equals if they have the same 
```

Comment:
Wouldn't this be slightly better wording (blatantly copied from [`java.time.LocalDate.equals`][1]): 
```ceylon
"Checks if this date is equal to another date.
 
 Compares this Date with another ensuring that the date both objects refer to is the same."
```

[1]:  https://docs.oracle.com/javase/8/docs/api/java/time/LocalDate.html#equals-java.lang.Object-